sed/loader/utils.py

```diff
@@ -0,0 +1,45 @@
+"""Utilities for loaders
+"""
+from glob import glob
+from typing import cast
+from typing import List
+
+from natsort import natsorted
+
+
+def gather_files(
```

Comment:
My proposal was to have these utility functions in the base class [even above the abstract class], including the other dataframe features we wanted for our dataframe class. But perhaps I can later implement that, as it should not change the functionality, but hopefully just make the plugin-development process simpler.

Reply:
Well, this is so far only a function to collect files from a folder. How to integrate the dataframe class Steiin has been working on is probably a separate topic.

Reply:
Right. I think that'll hopefully reveal itself with more loader modules.